pipelines/samples/signing-operator/README.md

```diff
@@ -0,0 +1,174 @@
+
+# About image signing operator
+
+The image signing opeator is for automating various configurations which are required enabling the image signing by the Tekton pipeline in Kanabero.
+The current code does following:
+- generate or import RSA keypair for the image signing based on the values of ImageSigning custom resource instance.
+- create the signature-secret-key secret resource from the RSA keypair. This secret is consued by the image signing task of Tekton pipeline.
+- When the ImageSignig custom resource is deleted, the corresponding signature-secret-key resource is also deleted.
+- If the generated signature-secret-key resource is deleted, the same secret will be created by the image signing operator.
+
+# Build and install the image signing operator
+
+## Prerequisite
+
+The image signing operator is developed using operator-sdk version 0.11.0.
+
+In order to build and deploy the operator, the following prerequisite packages need to be installed:
+- go version 1.13 or later
+- operator SDK version 1.11.0
+- Openshift Container Platform 4.2 CLI
+
+## Building the sample operator
+
+### Clone the image signing operator
+
+```
+git clone https://github.com/kabanero-io/kabanero-security/pipelines/samples/signing-operator
+cd signing-operator
+```
+
+You will need to examine the Makefile and set any necessary variables to push your container images to the correct repository. Especially, make sure that the variable of REPO is point to the corret repository, otherwise, the image will not pe pushed.
+
+### Login to OCP 
+(example)
+```
+oc login -u admin -p admin https://openshift.my.com:8443/
+```
+
+### Build image and push the image to the specified image repository
+
+```
+make build-image
+make push-image
+```
+
+### Deploy the operator and sets required resources.
+
+```
+make deploy
+```
+
+Make sure that the image signing operator is running.
+
+```
+oc get pods -n kabanero
+```
+(example command output)
+```
+[admin@openshift signing-operator]# oc get pods -n kabanero
+NAME                                                       READY   STATUS    RESTARTS   AGE
+signing-operator-77b489cf7f-n4d9z                          1/1     Running   0          49s
+```
+
+### Create a image signing custom resource instance for generating a keypair.
+
+If you have installed the image signing sample, please delete existing signature-secret-key secret, because the image signing operator will not create a secret if it already exists. To delete the secret, run
+```
+oc delete secret/signature-secret-key -n kabanero
+```
+If the image signing sample has not been installed, install it by using
+kabanero-security/pipelines/samples/signing/create-resources-for-signing-operator-4.sh
+This script is the same as the original scrept which configures image signing sample pipeline, but it skips generating a keypair. Make sure that the file is modified to set required parameters for your environment prior to run it.
```

Comment:
Change "scrept" to "script" and "prior to run it" to "prior to running it" .

Reply:
done.